woocommerce/wc-admin
# Fix for gutenberg 3.4

After updating to Gutenberg 3.4, `wc-admin` was resulting in some fatals. This branch is my attempt to fix that, and remove the deprecated usage of `withAPIData()`

__To Test__
- Update Gutenberg to 3.4
- Load up this branch and verify wp-admin will load, and wc-admin screens are functional.
